test/blocklib/core/fft/fft.hpp

```diff
@@ -156,15 +66,15 @@ struct fft : node<fft<T>> {
 
     constexpr DataSet<U>
     process_one(T input) noexcept {
-        if constexpr (std::is_same_v<T, InHistoryType>) {
+        if constexpr (std::is_same_v<T, InDataType>) {
```

Comment:
I think this is a good if not __the__ use case for using process_bulk() and the input spans.... The history buffer would be redundant with that.

Reply:
I agree, process_bulk() probably is a better option and does not require history buffer. Probably this would be the way to go.